example-aet-swarm/src/environment/docker-compose.yml.peb

```diff
@@ -38,7 +38,7 @@ services:
         constraints:
           - node.role == manager
     networks:
-      - private
+      - docker-net
```

Comment:
this name is simply GEP default; so to fit the convention I decided to rename it